components/cap-map.js

```diff
@@ -3,7 +3,7 @@ import { baseStyles } from "../lib/wc-base.js";
 import "./accessible-map.js";
 import { mapAbbreviations, mapData } from "../data/map.js";
 
-export class CaselawMap extends LitElement {
+export class CapMap extends LitElement {
```

Comment:
Renamed this so that we don't have a million "CaselawX" components — I think "CapMap" or "CapHeader" is probably less verbose